corehq/apps/translations/app_translations/download.py

```diff
@@ -224,6 +225,18 @@ def get_module_search_command_rows(langs, module):
     ]
 
 
+def get_case_search_rows(langs, module):
```

Comment:
I don't know anything about this but isn't case search behind a feature flag? so this should all be behind a feature flag?

Reply:
Yeah, that's a good point, even thought if the feature flag is off it's unlikely there will be any properties.

I think I also missed checking for the feature flag when adding `get_module_search_command_rows`.

Reply:
Nevermind about `get_module_search_command_rows`, Sravan already fixed it in an earlier PR.

Reply:
Ya, it just reduces computation by doing an early return